client/apps/shipping-label/view.js

```diff
@@ -6,6 +6,7 @@ import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import Gridicon from 'gridicons';
 import { translate as __ } from 'i18n-calypso';
+import { filter } from 'lodash';
```

Comment:
I haven't caught this in previous reviews (sorry!) but I thought we determined that we could load all of lodash without increasing the bundle size?